MvvmLight.AppCompat/packages.config

```diff
@@ -2,7 +2,6 @@
 <packages>
   <package id="CommonServiceLocator" version="1.3" targetFramework="monoandroid60" />
   <package id="MvvmLightLibs" version="5.2.0.0" targetFramework="monoandroid60" />
-  <package id="NuGet.CommandLine" version="3.3.0" targetFramework="monoandroid50" />
```

Comment:
Good spot - not required I just added it to get nuget.exe to create the nuget package